src/main/java/creativei/entity/Inquiry.java

```diff
@@ -22,14 +23,14 @@ public Inquiry(){}
 
     public Inquiry(InquiryVo inquiryVo) throws ParseException {
         this.setId(inquiryVo.getId());
-        this.name=inquiryVo.getName()==null||inquiryVo.getName().isEmpty()?null:inquiryVo.getName();
+        this.name= StringUtil.validateAndSetVo(inquiryVo.getName());
         this.areaOfInterest=AreaOfInterest.stringToEnum(inquiryVo.getAreaOfInterest());
-        this.phoneNumber=inquiryVo.getMobile()==null||inquiryVo.getMobile().isEmpty()?null:inquiryVo.getMobile();
-        this.email=inquiryVo.getEmail()==null||inquiryVo.getEmail().isEmpty()?null:inquiryVo.getEmail();
+        this.phoneNumber=StringUtil.validateAndSetVo(inquiryVo.getMobile());
+        this.email=StringUtil.validateAndSetVo(inquiryVo.getEmail());
```

Comment:
StringUtil should only have methods to manipulate or validate strings. Another working should be done here.

Reply:
Another working for email? It is also a string in vo and this method is used to validate if string is null or empty it returns null otherwise the string

Reply:
working of method is to validate the incoming vo name has been changed in commit 1b0438d please review.